core/go/build.gradle

```
@@ -28,6 +28,7 @@ ext {
         include "internal/**/*.go"
         include "pkg/**/*.go"
         include "componenttest/**/*.go"
+        include "noderuntests/**/*.go"
```

```suggestion
        include "noderuntests/**/*.go"
```

`componenttest/` has been removed